### ISSUE TITLE
Able to run, not able to use any commands

### ISSUE THREAD

**Issue by mtechtruc** [foolmoron/MafiaBot#2]
Discord.js: 7.0.1
Node: v7.1.0

Two types of error

running 

node --harmony_rest_parameters mafia-release.js
cause 
node: bad option: --harmony_rest_parameters

running
node mafia.js

works but running commands throw an error

(node:9380) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): Error: Unauthorized
(node:9380) DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:9380) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 2): Error: Unauthorized

Read the Readme that installing the bot doesn't work right now but thought I'd share in case anyone runs into it.

**Comment by foolmoron** [comment]
Yeah Discord.js has a new version that totally changes the API, so I'm gonna have to rewrite a lot of the code to work with that. You could run the current version with an old version of Discord.js, but it doesn't work on new versions of Node for several reasons.

**Comment by mtechtruc** [comment]
I do have an old version of Discord.js but would I need an older version of Node as well?
